packages/client/src/routers/SSRRouter.tsx

```diff
@@ -0,0 +1,149 @@
+import {Route, Routes} from 'react-router-dom';
+import {PATHS} from 'client/src/routers/name';
+import {AuthRoute} from './Routes/AuthRoute';
+import {GuestRoute} from './Routes/GuestRoute';
+import {ErrorBoundary} from 'client/src/components/ErrorBoundary';
+import {PageLanding} from 'client/src/pages/Landing';
+import {MainMenu} from 'client/src/pages/MainMenu';
+import {MainMenuMain} from 'client/src/pages/MainMenu/pages/main';
+import {MainMenuPlayAgainstAI} from 'client/src/pages/MainMenu/pages/playAgainstAI';
+import {MainMenuPlayOnline} from 'client/src/pages/MainMenu/pages/playOnline';
+import {MainMenuSettings} from 'client/src/pages/MainMenu/pages/settings';
+import {TestPage} from 'client/src/pages/TestPage';
+import {Page500} from 'client/src/pages/500';
+import {Page404} from 'client/src/pages/404';
+import {PageRegister} from 'client/src/pages/Register';
+import {Forum} from 'client/src/pages/Forum';
+import {ForumTopic} from 'client/src/pages/Forum/ForumTopic';
+import {ForumBox} from 'client/src/pages/Forum/compontents/ForumBox';
+import {PageAuth} from 'client/src/pages/Auth';
+import {PageProfile} from 'client/src/pages/Profile';
+import {PageProfileChangePassword} from 'client/src/pages/ProfileChangePassword';
+import {ForumDiscussion} from 'client/src/pages/Forum/ForumDiscussion';
+import {GameCanvasPage} from 'client/src/pages/GameCanvas';
+import {PageSetShips} from 'client/src/pages/Game/PageSetShips/PageSetShips';
+import {PageGame} from 'client/src/pages/Game/PageGame/PageGame';
+import {PageGameResults} from 'client/src/pages/Game/PageGameResults/PageGameResults';
+import {PageGameMechanicsDemonstration} from 'client/src/pages/GameMechanicsDemonstration';
+import {LeaderboardPage} from 'client/src/pages/Leaderboard';
+
+export const SSRRouter = () => {
+	return (
+		<Routes>
+			<Route
+				errorElement={<ErrorBoundary/>}>
+				<Route
+					path={PATHS.home}
+					element={<PageLanding/>}
+				/>
+				<Route
+					path={PATHS.forum}
+					element={<Forum/>}
+				>
+					<Route
+						path={PATHS.forum}
+						element={<ForumBox/>}
+					/>
+					<Route
+						path={':disccussionTitle'}
```

Comment:
```suggestion
						path={':discussionTitle'}
```
кажется, я тут в свое время опечатался. Нужно будет внести в задачу по рефакторингу

Reply:
Да, можно в рефакторинге сделать всё. Главное - сделать ту задачу по рефакторингу)